### PR TITLE
FlushService: Standardize on blob over filename

### PR DESCRIPTION
**Pull request by jdcaperon**
Small suggestion to standardize on using "blob" over "file" in the FlushService code to improve readability. Feel free to LBTM.

### CODE REVIEW

File: src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java [hunk 1]
```diff
@@ -504,44 +504,48 @@ private boolean shouldStopProcessing(
...
-  BlobMetadata buildAndUpload(String filePath, List<List<ChannelData<T>>> blobData)
-      throws IOException, NoSuchAlgorithmException, InvalidAlgorithmParameterException,
-          NoSuchPaddingException, IllegalBlockSizeException, BadPaddingException,
+  BlobMetadata buildAndUpload(String blobPath, List<List<ChannelData<T>>> blobData)
+      throws IOException,
+          NoSuchAlgorithmException,
+          InvalidAlgorithmParameterException,
+          NoSuchPaddingException,
+          IllegalBlockSizeException,
+          BadPaddingException,
```
Comment (jdcaperon) [inline src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java:520]: Seems my GJF doesn't like the old formatting :/

**Review by sfc-gh-tzhang — APPROVED** [review]
lgtm!

**Comment by sfc-gh-tzhang** [comment]
@sfc-gh-japatel @sfc-gh-rcheng any idea on the merge gate is failing here for external contributors?

**Comment by sfc-gh-rcheng** [comment]
> @sfc-gh-japatel @sfc-gh-rcheng any idea why the merge gate is failing here for external contributors?

Those tests run against our cloud deployments which external contributors don't have the access keys for. In this past KC [PR](https://github.com/snowflakedb/snowflake-kafka-connector/pull/656), I just opened a PR for them. We can do this again but long term we should share those keys through github (if possible, i think @sfc-gh-japatel tried at one point?)

**Pull request closed without merging**